src/theory/quantifiers/macros.cpp

```diff
@@ -21,6 +21,7 @@
 #include "options/quantifiers_modes.h"
 #include "options/quantifiers_options.h"
 #include "proof/proof_manager.h"
+#include "smt/smt_engine.h"
```

Comment:
Including smt_engine.h is now required for it to compile because smt_engine_scope.h only includes a forward declaration of SmtEngine.